src/components/ui/carousel.tsx

```
@@ -0,0 +1,262 @@
+    React.useEffect(() => {
+      if (!api || !setApi) {
+        return
+      }
+
+      setApi(api)
+    }, [api, setApi])
+
+    React.useEffect(() => {
+      if (!api) {
+        return
+      }
```

ESLint

[error] Missing semicolon.

(semi)